src/components/Tag/Tag.tsx

```diff
@@ -33,15 +35,17 @@ type TagPropsSelected = {
     type: TagType.Selected | TagType.SelectedWithFocus;
     label: string;
     onClick?: (event?: MouseEvent<HTMLButtonElement>) => void;
+    size?: TagSize;
 };
 
 type TagPropsUnselected = {
     type: TagType.Suggested | TagType.PreviouslySelected;
     label: string;
     onClick?: null;
+    size?: TagSize;
 };
 
-export const Tag: FC<TagProps> = ({ type, label, onClick }) => {
+export const Tag: FC<TagProps> = ({ type, label, onClick, size = 'm' }) => {
```

Comment:
same here -> `size = Tagsize.Medium`

Reply:
Got it, fixed for all occurences! :)

Reply:
Amazing! Thank you so much 🙌🏼 

That would be great if you could remove the FC, same discussion here: https://weare.frontify.com/document/42#/frontend-react/typescript/stop-using-react-fc

We should take the habit to remove it in all components we update if we see it still using it. So we can get this all cleaned up overtime 😄

Reply:
Sure, makes sense! Done!